tests/test_theoretical_property.py

```diff
@@ -20,7 +32,7 @@ def test_that_update_is_according_to_theory():
     """
     N = 1500
     nparam = 3
-    var = 2
+    var = 2.0
```

Comment:
why...

Reply:
Just checking that you are paying attention.
Fixed 👍